Kickstarter-iOS/Views/SelectCurrencyTableViewHeader.swift

```diff
@@ -0,0 +1,63 @@
+import Library
+import Prelude
+import Prelude_UIKit
+import UIKit
+
+final class SelectCurrencyTableViewHeader: UITableViewHeaderFooterView {
+
+  override init(reuseIdentifier: String?) {
+    super.init(reuseIdentifier: reuseIdentifier)
+
+    self.addSubview(self.headerStackView)
+    self.headerStackView.constrainEdges(to: self)
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.headerImageView
+      |> \.contentMode .~ .scaleAspectFill
+
+    _ = self.headerStackView
+      |> \.axis .~ .vertical
+      |> \.alignment .~ .center
+      |> \.spacing .~ Styles.grid(2)
+      |> \.layoutMargins .~ .init(
+        top: Styles.grid(4), left: Styles.grid(2), bottom: Styles.grid(2), right: Styles.grid(2)
+      )
+      |> \.isLayoutMarginsRelativeArrangement .~ true
+
+    _ = self.headerLabel
+      |> settingsDescriptionLabelStyle
+      |> \.textColor .~ .ksr_text_dark_grey_500
+  }
+
+  // MARK: Accessors
+
+  public var text: String? {
+    didSet {
+      self.headerLabel.text = text
+    }
+  }
+
+  // MARK: Subviews
+
+  private lazy var headerStackView: UIStackView = {
+    let stackView = UIStackView(arrangedSubviews: [
```

Comment:
Is there a need for defining local variable as oppose to returning `UIStackView` directly just like we do in `headerImageView` and other places?

Reply:
There is no need!